### PR TITLE
[ticket/13313] Add event core.acp_email_display

### PR DESCRIPTION
**Pull request by Elsensee**
https://tracker.phpbb.com/browse/PHPBB3-13313

### CODE REVIEW

**Comment by Elsensee** [comment]
Wait.. should this go into 3.1.4 now?

File: phpBB/includes/acp/acp_email.php [hunk 1]
```diff
@@ -323,8 +323,22 @@ function main($id, $mode)
...
 
+		/**
+		* Modify custom email template data before we display the form
+		*
+		* @event core.acp_email_display
+		* @var	array	template_data		Array with template data assigned to email template
+		* @var	array	exclude				Array with groups which are excluded from group selection
+		* @var	array	usernames			Usernames which will be displayed in form
+		*
+		* @since 3.1.3-RC1
```
Comment (marc1706) [inline phpBB/includes/acp/acp_email.php]: Should be 3.1.4-RC1

Comment (Elsensee) [inline phpBB/includes/acp/acp_email.php]: Yup.. that's why I asked if this goes into 3.1.4 now.. but thanks for the answer ;)

Comment (Elsensee) [inline phpBB/includes/acp/acp_email.php]: Updated.

**Pull request merged**